docs/wip_glassbox/wip-glassbox-customization.md

```diff
@@ -1,5 +1,9 @@
 # WIP Glassbox Customization
 
+## Entities
+* visibility of all private properties and methods of DataFactories was changed to protected (@Miroslav-Stopka)
```

Comment:
Please move this point down and remove Entities section. Now we have duplicated Entities section

Reply:
Done